src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,250 @@
+- The `</p>` represents where the element ends.
+
+#### Common HTML Elements For UX Writers
+
+Use the strong and emphasis elements, to ensure users can perceive the semantic meaning indicated by the visual treatment, even if they’re using assistive technology like a screen reader.
+
+- The **strong element** `<strong></strong>`, defines text with strong importance.
+- The **emphasis element** `<em></em>`, defines text with stressed emphasis.
+
+> Use the **bold element** `<b></b>` and the **italic or idiomatic text element** `<i></i>` for formatting and to display special types of text.
+
+- The **bold element** `<b></b>` , draws attention to the element's contents, without extra importance.
+
+- The **italic or idiomatic text element** `<i></i>`, defines a part of text in an alternate voice or mood.
+
+<span id="semantic-elements" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h2"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayTwo"
+>
+  Semantic Elements
+</SprkHeading>
+
+Using HTML Semantic elements ensure users can perceive the semantic meaning indicated by the visual treatment, even if they’re using assistive technology like a screen reader.
+
+- [The Strong Element](#the-strong-element)
+- [The Emphasis Element](#the-emphasis-element)
+
+> To view a list of all the HTML elements available to add semantic meaning to copy, refer to [MDN’s Semantic element list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element#inline_text_semantics).
+
+### The Strong Element
+
+The <strong>strong element</strong> indicates strong importance, seriousness, or urgency of the enclosed text inside the `<strong></strong>` tags.
+
+- By default web browsers, render the contents inside strong tags in **bold type**.
+
+#### Usage
+
+Use for:
+
+- Referencing the name of a page
+- Representing strong importance of it's contents
+- Adding importance or urgency to pieces of a sentence
+
+<span id="examples" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h5"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayFive"
+>
+  Examples
+</SprkHeading>
+
+Here are some examples of emphasized word(s) using the strong element.
+
+- Check your **To Do List** items for new tasks.
+  - Check your `<strong> To Do List </strong>` items for new tasks.
+- You can change your communication preferences on **My Profile**.
+  - You can change your communication preferences on `<strong>My Profile</strong>`.
+- **Your payment is due** on September 22, 2021.
+  - `<strong>Your payment is due</strong>` on September 22, 2021.
+- Remember to bring your **driver’s license** to your closing appointment.
+  - Remember to bring your `<strong>driver’s license</strong>` to your closing appointment.
+
+#### The Strong vs The Bold Element vs Styling Text
+
+- Use the **strong element** `<strong></strong>` for important content, and to communicate urgency.
```

Comment:
`for important content` implies that anything that's important needs to be in a strong tag.

Let's change it to something like:
`Use the strong element to communicate importance or urgency`